flow/duration.yaml

```diff
@@ -104,6 +104,12 @@ components:
         A continuous burst of packets that will not automatically stop.
```

Comment:
please update this descriptions since this container will now support both continuous and fixed bursts.

Reply:
Done.